src/sqlfluff/core/rules/doc_decorators.py

```diff
@@ -51,7 +51,7 @@ def document_configuration(cls, ruleset="std"):
                 keyword, info_dict["definition"]
             )
             if "validation" in info_dict:
-                config_doc += " Must be one of {}.".format(info_dict["validation"])
+                config_doc += " Must be one of ``{}``.".format(info_dict["validation"])
```

Comment:
for my reference, are double backticks like the single backtick code blocks in markdown or something different?

Reply:
Yup. Had to look this one up myself to be sure:

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html

Reply:
Just learned this myself today!